samples/openthread/ncp/README.rst

```diff
@@ -0,0 +1,88 @@
+.. _ot_ncp_sample:
+
+Thread: NCP sample
+##################
+
+The Thread NCP sample demonstrates the usage of OpenThread Network Co-Processor architecture inside the Zephyr shell.
+
+Overview
+********
+
+The sample demonstrates usage of NCP target on the MCU to communicate it with user WPAN Network Daemon, allowing to create for example Thread Border Router.
+It is important to mention that, according to NCP architecture, MCU part needs to cooperate with user higher layer process to altogether establish complete Thread Border Router.
+Recommended solution is to use docker version of OTBR (OpenThread Border Router) and WPAN Network Daemon tool (wpantund), working under Linux operating system, for such purpose (for more information go there: `OpenThread Border Router`_).
+
+There is possibility to configure number of OpenThread features, but by default application has COMMISSIONER, JOINER, BORDER_ROUTER, BORDER_AGENT and DHCP6_SERVER functionalities enabled.
+
+Requirements
+************
+
+The sample supports the following development kits for testing the network status:
+
+.. include:: /includes/boardname_tables/sample_boardnames.txt
+   :start-after: set8_start
+   :end-before: set8_end
+
+Border Router higher layer process is needed to communicate with MCU border router part.
+Optionally, one or more compatible development kits programmed with OpenThread samples to test communication in bigger network.
+
+User interface
+**************
+
+Main recommended interaction with the application should be done via wpantund Linux process using its comannds (for more information check: `wpantund usage`_).
```

Comment:
"Recommended" can suggest some interactions are not recommended or discouraged. If so, we need to be more specific here.

Reply:
Done